core/src/code/mod.rs

```diff
@@ -53,6 +60,8 @@ pub struct Code {
     stack_end: Option<WasmPage>,
     /// Instruction weights version.
     instruction_weights_version: u32,
+    /// Section sizes used for charging during module instantiation.
```

Comment:
@grishasobol wdyt removing this until unified meta storage is implemented for codes? (will publish proposal soon) and operate only in terms of a field of `InstrumentedCode` which will be lazy re-calculated

Otherwise migrations within the PR will be really huge